app/src/main/java/com/celzero/bravedns/ui/AboutFragment.kt

```diff
@@ -17,135 +17,112 @@ package com.celzero.bravedns.ui
 
 import android.content.Intent
 import android.content.pm.PackageManager
-import android.net.Uri
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import android.widget.ImageView
-import android.widget.TextView
 import androidx.appcompat.app.AlertDialog
+import androidx.core.net.toUri
 import androidx.fragment.app.Fragment
 import com.celzero.bravedns.R
+import com.celzero.bravedns.databinding.FragmentAboutBinding
 import com.celzero.bravedns.service.AppUpdater
 import org.koin.android.ext.android.inject
 
 
 class AboutFragment : Fragment(), View.OnClickListener {
-
-    private lateinit var websiteTxt : TextView
-    private lateinit var twitterTxt : TextView
-    private lateinit var githubTxt : TextView
-    private lateinit var  blogTxt : TextView
-    private lateinit var mailTxt : TextView
-    private lateinit var telegramTxt : TextView
-    private lateinit var faqTxt : TextView
-    private lateinit var mozillaImg : ImageView
-    private lateinit var appVersionText : TextView
-    private lateinit var appUpdateTxt : TextView
-    private lateinit var whatsNewTxt : TextView
+    private var _binding: FragmentAboutBinding? = null
+    private val b get() = _binding!!
```

Comment:
Is there a reason you are not using `private **lateinit** var _binding` instead of this forced non-null workaround?

Reply:
Yes, to avoid memory leaks.
https://proandroiddev.com/avoiding-memory-leaks-when-using-data-binding-and-view-binding-3b91d571c150

Reply:
Cool, I did not know that. The article also mentions the `viewBinding()` delegate which imo looks cleaner and wouldn't confuse someone who didn't know that like it did with me.

Reply:
Probably best to add a code comment, @HrBDev